samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.twitter_client import TwitterClient
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
```

Comment:
could we move these into the twitter util package? the point is that the user's not supposed to focus on this stuff